config/routes.rb

```diff
@@ -24,7 +24,8 @@
       get :part, on: :new
     end
 
-    resources :recommendations, only: %i[new create]
+    resources :recommendations, only: %i[new create update]
```

Comment:
No because `resources` requires an :id for edit whereas `resource` does not and since we don't have render the page for a single resource.

I want the url like `/recommendations/edit` rather than ``/recommendations/edit/:id` otherwise will have to load up a query for recommendations on the planning application show page to find this for providing the url

Reply:
Ohh...yes, I get you 😄